src/api/todos.ts

```diff
@@ -0,0 +1,22 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const postTodos = (userId: number, title: string) => {
+  return client.post<Todo>(`/todos?userId=${userId}`, {
+    userId,
+    title,
+    completed: false,
+  });
+};
+
+export const deleteTodos = (id: number) => {
+  return client.delete(`/todos/${id}`);
+};
+
+export const patchTodos = (id: number, data: Partial<Todo>) => {
```

Comment:
The function name 'patchTodos' should be more descriptive. Consider renaming it to 'updateTodo'.